build.gradle

```diff
@@ -471,6 +557,26 @@ project(':iceberg-mr') {
       include 'NOTICE'
     }
 
+    dependencies {
+      exclude (dependency('org.apache.iceberg:iceberg-parquet'))
```

Comment:
As above for all these. Perhaps you're having to do this due to transitive dependencies?

Reply:
Yeah a lot of these are exactly because of that - shading everything completely throws off those pre-build checks for the runtime uniqueness. I'll definitely have a quick sweep over the file to get rid of some of the exclusions that have no effect

Reply:
@massdosage i've gone through and removed a bunch of exclusions that didn't need to be included, i will hold of alphabetising them until we decide what the guava solution is :')

Reply:
Looks better, I've still got some questions about some of the ones above, mainly to do with iceberg projects that depend on each other.